config/settings/base.py

```diff
@@ -377,8 +377,12 @@
     "OSLER_ATTENDANCE_URL",
     default="https://www.wustl.edu",
 )
+OSLER_ABOUT_URL = env(
```

Comment:
Having this as an environmental variable which can also be overridden seems redundant, but that's not a big issue